scipy/stats/_continuous_distns.py

```diff
@@ -5025,11 +5025,9 @@ class lognorm_gen(rv_continuous):
 
     %(after_notes)s
 
-    A common parametrization for a lognormal random variable ``Y`` is in
-    terms of the mean, ``mu``, and standard deviation, ``sigma``, of the
-    unique normally distributed random variable ``X`` such that exp(X) = Y.
-    This parametrization corresponds to setting ``s = sigma`` and ``scale =
-    exp(mu)``.
+    Suppose a normally distributed random variable ``X`` has  mean ``mu`` and
+    standard deviation ``sigma``. Then ``Y = exp(X)`` is lognormally
+    distributed with ``s = sigma`` and ``scale = exp(mu)``.
```

Comment:
Another possibility, which I think implements @rkern's suggestion more strictly:
```suggestion
    If ``X`` is normally distributed, then ``Y = exp(X)`` is lognormally
    distributed. In this case, if ``X`` has  mean ``mu`` and standard
    deviation ``sigma``, then ``Y`` is parameterized by ``s = sigma`` and
    ``scale = exp(mu)``.
```